src/script/ui.ts

```diff
@@ -0,0 +1,26 @@
+import { Post } from "./api";
```

Comment:
We can use `type` here to optimize this. 
```suggestion
import type { Post } from "./api";
```

Reply:
add https://github.com/Sakshi-Simform/product-management/pull/3/commits/9c5516df96f609dd01b52a749d2f3637e6cff67b